src/components/v5/common/ActionSidebar/partials/UserSelect/UserSelect.tsx

```diff
@@ -52,16 +61,26 @@ const UserSelect: FC<UserSelectProps> = ({ name }) => {
           },
         )}
         onClick={toggleUserSelect}
-        aria-label={formatMessage({ id: 'ariaLabel.selectUser' })}
+        aria-label={formatText({ id: 'ariaLabel.selectUser' })}
       >
-        {userByAddress ? (
-          <UserAvatar
-            user={userByAddress}
-            userName={userDisplayName}
-            size="xs"
-          />
+        {field.value ? (
+          <>
+            <UserAvatar
+              user={userByName || userByAddress}
+              userName={userDisplayName}
+              size="xs"
+              className={
+                usersOptions.isRecipientNotVerified ? 'text-warning-400' : ''
+              }
+            />
+            {usersOptions.isUserVerified && (
+              <span className="flex ml-2 text-blue-400">
+                <Icon name="verified" />
+              </span>
+            )}
+          </>
```

Comment:
What if `field.value` is set but the user is not found?

Reply:
Not sure what you mean by that, can you show me an example of how that looks when `field.value` is set and `userByName` and `userByAddress` is undefined?